article-analyzer/src/main/java/com/reader/article_analyzer/ArticleCategoryAnalyzer.java

```diff
@@ -0,0 +1,68 @@
+package com.reader.article_analyzer;
+
+import com.reader.article_analyzer.Model.Article;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+import java.io.IOException;
+import java.util.*;
+
+@Component
+public class ArticleCategoryAnalyzer {
+    private final JsonFileService fileService;
+    private final String[] excludedWordsArray;
+    private final List<Set<String>> allCategories;
+
+    public ArticleCategoryAnalyzer(@Value("${excluded.words}") String excludedWords, JsonFileService fileService) {
+        this.excludedWordsArray = excludedWords.split(", ");
+        this.fileService = fileService;
+        this.allCategories = new ArrayList<>(); // Ініціалізуємо список категорій
```

Comment:
this comment tells the same that the code says to us, remove it

![image](https://github.com/vitaliiStefanchakInventorsoft/spring-lecture/assets/103140969/a0afc584-f91c-4fba-8d44-14129af675f4)

Reply:
Changed it